### PR TITLE
adding date to log entries

### PR DESCRIPTION
**Pull request by monikavar**
Adding timestamp in UTC in log template.

### CODE REVIEW

**Review by barnstee — APPROVED** [review]

File: common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/Extensions/SerilogEx.cs [hunk 1]
```diff
@@ -66,6 +66,6 @@ public static class SerilogEx {
         }
 
         private const string kDefaultTemplate =
-            "[{Timestamp:HH:mm:ss} {Level:u3} {SourceContext:lj}] {Message:lj} {NewLine}{Exception}";
+            "[{Timestamp:yyyy-MM-dd HH:mm:ss} {Level:u3} {SourceContext:lj}] {Message:lj} {NewLine}{Exception}";
```
Comment (karok2m) [inline common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/Extensions/SerilogEx.cs:69]: @monikavar, can the time zone also be added ? UTC would be preferable.

Comment (monikavar) [reply common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/Extensions/SerilogEx.cs:69]: Good point!
What would be more preferable? UTC time or local time with timezone?
e.g.,
1. `2020-09-07 14:22:29Z`
2. `2020-09-07 16:22:29 +02:00`

Comment (karok2m) [reply common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/Extensions/SerilogEx.cs:69]: I would prefer UTC one.

Comment (monikavar) [reply common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/Extensions/SerilogEx.cs]: done.

**Review by cristipogacean — APPROVED** [review]

**Review by karok2m — APPROVED** [review]

**Pull request merged**